src/components/NFTPortfolio/State.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-  <div>
+  <component :is="tag">
```

Comment:
need to add `tag` to props

Reply:
Seems I added changes in the wrong [commit](https://github.com/likecoin/liker-land/pull/556/commits/471e0f90efa65932d9ac71252f8c35521189d21b#diff-e360e555023d5c11dcb5a54cdab861b6ab9aafc4f54de2fa1b953463c44356edR16-R19) 😅
Sorry for being confusing.
I will resubmit commits.